pom.xml

```diff
@@ -247,9 +248,9 @@
             </exclusions>
          </dependency>
          <dependency>
-            <groupId>io.admin-shell.aas</groupId>
+            <groupId>org.eclipse.digitaltwin.aas4j</groupId>
```

Comment:
Are you sure it's safe to directly switch over to aas4j-dependencies instead of adding them additionally? It could cause some inconveniences for projects, that want to use the latest `esmf-parent` but still stay on the admin-shell.io components (i.e. they have to explicitely add versions for the old dependencies or use their own dependency management for them). If this is not given and it's "all or nothing" anyway, you can of course keep it like that.

Reply:
Usually I'd agree, but since in the ESMF Java universe the esmf-sdk's esmf-aspect-model-aas-generator is the only component that uses this dependency (an will likely be switched after the parent upgrade) I say go with it.